demos/shell.yaml

```diff
@@ -64,12 +64,13 @@ spec:
               Exit the shell with an exit code 1
             serialization:
             payload: |
-              exit 1
+              ls ?
+              exit $?
         - recv:
             doc: |
               Handle the shell exit code 1
             topic: exit
-            regexp: 1
+            regexp: "2"
```

Comment:
I get a failure on this line when `regexp` is set to `2`

Reply:
This should be fixed